code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -81,8 +81,12 @@
 	if(!(obscured & ITEM_SLOT_EYES) )
 		if(glasses  && !(glasses.item_flags & EXAMINE_SKIP))
 			. += "[t_He] [t_has] [glasses.get_examine_string(user)] covering [t_his] eyes."
-		else if(eye_color == BLOODCULT_EYE && IS_CULTIST(src) && HAS_TRAIT(src, CULT_EYES))
+		else if(eye_color == BLOODCULT_EYE && IS_CULTIST(src) && HAS_TRAIT(src, TRAIT_CULT_EYES))
 			. += "<span class='warning'><B>[t_His] eyes are glowing an unnatural red!</B></span>"
+		else if(eye_color == BLOODCULT_EYE && HAS_TRAIT(src, TRAIT_BLOODSHOT_EYES))
+			. += "<span class='warning'><B>[t_His] eyes are bloodshot!</B></span>"
+		else if(eye_color == BLOODCULT_EYE)
+			. += "<span class='warning'><B>[t_His] eyes are an unnatural red!</B></span>"
```

Comment:
the way youre doing this check is bad

Reply:
Instead of repeating the `eye_color == BLOODCULT_EYE` check three times, do it once and nest the rest inside of it:
```dm
else if(eye_color == BLOODCULT_EYE)
	if(S_CULTIST(src) && HAS_TRAIT(src, TRAIT_CULT_EYES))
		. += "<span class='warning'><B>[t_His] eyes are glowing an unnatural red!</B></span>"
	else if(HAS_TRAIT(src, TRAIT_BLOODSHOT_EYES))
		. += "<span class='warning'><B>[t_His] eyes are bloodshot!</B></span>"
```
I don't think you want the last one, as that'd change behavior.